src/components/HotRepositories.tsx

```diff
@@ -74,21 +74,21 @@ const HotRepositories = ({ user }: HotReposProps): JSX.Element => {
       <div className="grid xs:grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-4 w-full my-5">
         {hotRepos.map(
           ({ id, full_name, name, description, issues, stars, contributions }) => (
-            <div key={id} className="p-4 border rounded-lg bg-white w-full space-y-1 relative">
+            <div key={id} className="p-4 border rounded-2xl bg-white w-full space-y-1 relative">
               {/* header & upvote button */}
               <div className="flex justify-between w-full">
                 <div className="flex space-x-1 items-center">
                   <img src={getAvatarLink(full_name.replace(`/${name}`, ''))} alt="Hot Repo Icon" className="h-4 w-4 rounded-md overflow-hidden" />
-                  <span className="text-xs text-gray-400">{full_name.replace(`/${name}`, '')}</span>
+                  <span className="text-sm font-medium text-lightSlate11">{full_name.replace(`/${name}`, '')}</span>
                 </div>
                 <button
-                  className={`px-2 py-0.5 border rounded-lg flex justify-center items-center space-x-1 text-xs transition-all duration-200 ${
-                    checkVoted(id) ? "text-saucyRed border-saucyRed " : "text-grey border-gray-500 "
+                  className={`px-2 py-0.5 border  rounded-lg flex justify-center items-center space-x-1 text-xs transition-all duration-200 ${checkVoted(id)?'bg-lightOrange01':'bg-lightSlate01'} ${
```

Comment:
If [this PR](https://github.com/open-sauced/hot/pull/269) merged before getting approval to do merge run `npm run format` "Hope it catches the withespaces"  if not please delete the empty spaces